finder-app/c.txt

```diff
@@ -0,0 +1 @@
+Hello
```

Comment:
@midhun9 these files could probably be delted, don't need to be added to your repo